server/repository/src/db_diesel/insurance_provider_row.rs

```diff
@@ -63,6 +63,15 @@ impl<'a> InsuranceProviderRowRepository<'a> {
         Ok(result)
     }
 
+    pub fn find_all_active(&self) -> Result<Vec<InsuranceProviderRow>, RepositoryError> {
+        let result = insurance_provider::table
+            .load(self.connection.lock().connection())?
+            .into_iter()
+            .filter(|row: &InsuranceProviderRow| row.is_active == true)
+            .collect();
+        Ok(result)
+    }
```

Comment:
```suggestion
    pub fn find_all_active(&self) -> Result<Vec<InsuranceProviderRow>, RepositoryError> {
        let result = insurance_provider::table
            .filter(insurance_provider::is_active.eq(true))
            .load(self.connection.lock().connection())?;
        Ok(result)
    }
```

This will update the sql query so the database does the filtering.
In this case it's probably not very important as there won't be a lot but I think it's still worth updating.